src/core/Menu/Menu.tsx

```diff
@@ -91,7 +109,35 @@ export const Menu = (props: MenuProps) => {
       ref={menuRef}
       {...rest}
     >
-      {children}
+      {React.Children.map(children, (item) => {
+        if (React.isValidElement(item)) {
+          const subItems = (item.props as MenuItemProps)?.subMenuItems ?? [];
+          if (subItems.length === 0) {
+            return item;
+          } else {
+            return (
+              <Popover
```

Comment:
Why handling submenu here and not in `MenuItem`?

Reply:
I've tried many different things, but I'm still struggling with focus